Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityControllerMappingProfile.cs

```diff
@@ -80,8 +80,9 @@ public GameObject GlobalRightHandModel
             new MixedRealityControllerMapping(8, "Oculus Touch Controller Left", typeof(OculusTouchController), Handedness.Left),
             new MixedRealityControllerMapping(9, "Oculus Touch Controller Right", typeof(OculusTouchController), Handedness.Right),
             new MixedRealityControllerMapping(10, "Oculus Remote Controller", typeof(OculusRemoteController)),
-            new MixedRealityControllerMapping(11, "Generic OpenVR Controller Left", typeof(GenericOpenVRController), Handedness.Left, true),
-            new MixedRealityControllerMapping(12, "Generic OpenVR Controller Right", typeof(GenericOpenVRController), Handedness.Right, true),
+            new MixedRealityControllerMapping(11, "Touch Screen Input", typeof(UnityTouchController), Handedness.Any),
```

Comment:
in the future, we should try to avoid reordering collections in profiles so that we don't create unwanted side effects

Reply:
Agreed, but there will be one more change with the addition to the mouse that is in the next PR. I want to look into making it easy to rearrange this in the future.